src/components/SkeletonThumbnail/SkeletonThumbnail.scss

```diff
@@ -9,13 +9,13 @@
 }
 
 .sizeSmall {
-  @include skeleton-thumbnail-size(rem(30px));
+  @include skeleton-thumbnail-size(rem(40px));
 }
 
 .sizeMedium {
-  @include skeleton-thumbnail-size(rem(40px));
+  @include skeleton-thumbnail-size(rem(60px));
 }
 
 .sizeLarge {
-  @include skeleton-thumbnail-size(rem(50px));
+  @include skeleton-thumbnail-size(rem(80px));
```

Comment:
Is there a way to import these from the actual `Thumbnail` component? Otherwise they're going to get out of sync again next time something changes

Reply:
Maybe a simple function? `thumbnailSize(small)`?

Reply:
Done! Let me know what y'all think 😀